build.sbt

```diff
@@ -217,9 +217,12 @@ lazy val V = new {
   val kindProjector         = "0.13.3"
   val coursierApi           = "2.1.9"
   val coursierInterface     = "1.0.19"
-  val scalameta             = "4.8.8"
+  val scalameta             = "4.9.1"
   val localSnapshotVersion  = "0.7.0-SNAPSHOT"
   // scala-steward:off
   val zio = "1.0.18"
   // scala-steward:on
 }
+
+// Scala steward dependencies for auto-update
+lazy val scalameta = "org.scalameta" %% "scalameta" % V.scalameta
```

Comment:
Is this enough to do? We don't have to add the dep to any project?

Reply:
I am not sure. It is probably safer to add it to a project then.